GPflowOpt/domain.py

```diff
@@ -46,6 +46,12 @@ def __add__(self, other):
         assert isinstance(other, Domain)
         return Domain(self._parameters + other._parameters)
 
+    def __radd__(self, other):
+        if other == 0:
+            return self
+
+        return other + self
+
```

Comment:
I avoided writing this method by using np.sum instead of sum(). Right now it causes a decrease in coverage, and it is difficult to test this.